homeassistant/components/geonetnz_quakes/__init__.py

```diff
@@ -96,6 +98,7 @@
     manager = hass.data[DOMAIN][FEED].pop(config_entry.entry_id)
     await manager.async_stop()
 
-    await hass.config_entries.async_forward_entry_unload(config_entry, "geo_location")
+    for domain in COMPONENTS:
+        await hass.config_entries.async_forward_entry_unload(config_entry, domain)
```

Comment:
If order is not important we should always leverage asyncio to await all tasks concurrently, eg by using `asyncio.wait`.

Reply:
OK. Order is indeed not important, changed to creating async tasks.